packages/fiori/src/Page.ts

```diff
@@ -36,26 +36,15 @@ import PageCss from "./generated/themes/Page.css.js";
  * that there is enough space for the <code>ui5-page</code> to be rendered.
  * <b>Note:</b> In order for the <code>ui5-page</code> to be displayed, the parent element should have fixed height.
```

Comment:
Let's add one <br /> tag here just to make the documentation to looks better.

Reply:
Done